client/my-sites/plans-v2/plans-filter-bar/style.scss

```diff
@@ -0,0 +1,75 @@
+.plans-filter-bar {
+	$height: 36px;
+	// SegmentedControl uses a border-radius of 4px while our design specifies 3px
```

Comment:
I'd say it's okay to use the border radius of the current component. It makes the styles leaner and the components more consistent across the app.

Reply:
Good advice. It's hard to find the right balance between being consistent with what we already have and following the designs.